elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -27,6 +27,74 @@ class PfeAutocomplete extends PFElement {
     return "pfe-autocomplete.scss";
   }
 
+  static get properties() {
+    return {
+      initValue: {
+        title: "Initial Value",
+        type: String,
+        observer: "_initValueChanged"
+      },
+      _initValue: {
```

Comment:
I like this underscore approach a lot for highlighting attributes to be deprecated.  Can we do that in accordion too and explicitly define the attr instead?

Reply:
I've been using the pattern `oldInitValue` instead.  Since `_` is a convention for "private", I wanted to use something else.  Whatever the pattern, I agree it should be used in accordion and everywhere.  Kyle and I have also been adding `// @TODO Deprecate property` alongside every old property def.

Reply:
old it is. I'll update any properties prefixed with an underscore.